docs/_data/portland-2022-schedule.yaml

```diff
@@ -0,0 +1,96 @@
+writing_day:
+  - time: '8:00 <a href="https://time.is/PT">Pacific</a>'
+    title: 'Platform opens'
+  - time: '9:00'
+    title: Morning introduction - Introduction to Sprints & Project Introductions
+  - time: '9:30'
+    title: Team up and start working
+  - time: '12:00'
+    title: Lunch Break
+  - time: '12:30'
+    title: Conference Intro and Tour with the Welcome Wagon
+  - time: '5:00'
+    title: End of Writing Day
+
+talks_day1:
+  - time: '8:00 <a href="https://time.is/PT">Pacific</a>'
+    title: Platform opens
+  - time: '8:30'
+    title: Welcome Wagon intro to WTD
+  - time: '9:00'
+    title: Write the Docs Team - Community update
+  - time: '9:30'
+    title: "<b>Unconference Starts</b>"
+  - time: '9:30'
+    slug: TBD 1/2
+  - time: '10:00'
+    title: '5 minute break'
+  - time: '10:05'
+    slug: TBD 2/2
+  - time: '10:35'
+    title: '5 minute break'
+  - time: '10:40'
+    title: 'Two talk Q&A'
+  - time: '10:55'
+    title: '5 minute break'
```

Comment:
If we extend this break to be the morning snack break it means that lunch can be pushed to 12:30 pretty safely imo... thoughts?